lib/fluent/plugin/out_google_cloud.rb

```diff
@@ -207,6 +206,24 @@ def initialize
       @log = $log # rubocop:disable Style/GlobalVars
     end
 
+    # Set up regexp patterns to parse tags and logs.
+    def setup_regexp_patterns
```

Comment:
Why did you add this before `configure`, when all of the other new functions are after? Does this function even need to be public?